landoapi/api/revisions.py

```diff
@@ -1,18 +1,38 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import os
 from connexion import problem
-
-
-def search():
-    pass
+from phabricator import Phabricator
 
 
 def get(id):
+    # Most often IDs are given in the form 'D12345', for example.
+    # The Phabricator Conduit APIs expect them to be given as integers.
+    id_num = id.strip().replace('D', '')
+    phab = get_phabricator_client()
+    response = phab.differential.query(ids=[id_num])
+    if response and response[0]:
```

Comment:
Let's call this "revisions"; response makes me think of a network request, which it may be, but seems like it could be confusing.